tests/test_config.py

```diff
@@ -9,39 +9,40 @@
 from betterconf.caster import to_int
 from betterconf.config import AbstractProvider, Field, as_dict
 from betterconf.config import VariableNotFoundError
+from betterconf.exceptions import ImpossibleToCastError
 
 VAR_1 = "hello"
 VAR_1_VALUE = "hello!#"
 
 
 class TestConfig(Config):
-    debug = field('DEBUG', default=False, caster=to_bool)
+    debug = field("DEBUG", default=False, caster=to_bool)
```

Comment:
I don't feel like you *should* do this - it's nothing more than a change that doesn't change anything from my point of view... And also this adds way too many changed lines! :)

Reply:
That's _black_, our code formatter. It's neither useless nor useful. It is.

Reply:
Oh, really? Sorry, didn't know that. Since when it was added?

Reply:
Well... Since [the beginning](https://github.com/fscdev/betterconf/blob/20005351741034e234fb38b299b2319711826fcb/pyproject.toml#L12)